psi4/driver/driver.py

```diff
@@ -1556,7 +1556,7 @@ def vibanal_wfn(wfn, hess=None, irrep=None, molecule=None, project_trans=True, p
 
     vibinfo, vibtext = qcdb.vib.harmonic_analysis(
         nmwhess, geom, m, wfn.basisset(), irrep_labels, dipder=dipder, project_trans=project_trans, project_rot=project_rot)
-    vibrec.update({k: qca.to_dict() for k, qca in vibinfo.items()})
+    vibrec.update({k: qca.json() for k, qca in vibinfo.items()})
```

Comment:
This builds serialized JSON blobs?

Reply:
yes, read back in [example here](https://github.com/psi4/psi4/pull/1654/files#diff-ef77f359ab40beab06fac4b16aa64518R110). it's awkward and not final vib schema form, but a step in the right direction.